src/vde_backend/retrieveDB.py

```diff
@@ -0,0 +1,221 @@
+__title__ = "retrieveDB"
+__version__ = "1.0.0"
+__author__ = "Alexandre Heneffe"
+__license__ = "MIT"
+
+
+""" 
+Script to save Flukso electrical data from the Cassandra database
+"""
+
+
+# standard library
+from datetime import timedelta
+import os
+import os.path
+import argparse
+
+# 3rd party packages
+import pandas as pd
+
+# local sources
+from constants import (
+	CASSANDRA_KEYSPACE, 
+	TBL_POWER
+)
+import pyToCassandra as ptc
+
+from utils import (
+	getDatesBetween, 
+	getLastRegisteredConfig
+)
+
+DATA_LOCAL_PATH = 'output/retrieved_data/' # TODO : put it in constants.py
+
+# ==============================================================================
+
+
+def getYesterday(now):
+	""" 
+	Given the timestamp of today, get the previous day's date.
+	-> YYYY-MM-DD
+	"""
+
+	yesterday = now.date()-timedelta(days=1)
+
+	return yesterday
+
+
+def saveDataToCsv(data_df, csv_filename, output_filename):
+	""" 
+	Save to csv
+	"""
+
+	if not os.path.exists(output_filename):
+		os.mkdir(output_filename)
+	filepath = os.path.join(output_filename, csv_filename)
+
+	data_df.to_csv(filepath)
+
+	print("Successfully Saved data in csv")	
+
+
+def getHomePowerDataFromCassandra(cassandra_session, home_id, date, table_name):
```

Comment:
Duplicate function. By the way, the `table_name` parameter seems useless because it should always be `TBL_POWER`.